Gemfile.lock

```diff
@@ -175,13 +175,13 @@ GEM
     childprocess (4.1.0)
     coderay (1.1.3)
     concurrent-ruby (1.1.10)
-    connection_pool (2.2.5)
+    connection_pool (2.3.0)
     crass (1.0.6)
-    cssbundling-rails (1.1.0)
+    cssbundling-rails (1.1.1)
```

Comment:
### [cssbundling-rails](https://github.com/rails/cssbundling-rails)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.1.0 → 1.1.1

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/cssbundling-rails/1.1.0/1.1.1)_]





<details>
<summary>Commits</summary>

A change of **7** commits. See the full changes on [the compare page](https://github.com/rails/cssbundling-rails/compare/v1.1.0...v1.1.1).

These are the individual commits:
- (c727896) [Change sass cli params to permit multiple inputs](https://github.com/rails/cssbundling-rails/commit/c727896b10f94beab394174d566e5a1f43f6172f)
- (d435ad1) [Merge pull request #83 from acallaghan/main](https://github.com/rails/cssbundling-rails/commit/d435ad15352da2a2b3df08056214f2eb9e4fbc72)
- (b451c60) [Use bundle add instead (#85)](https://github.com/rails/cssbundling-rails/commit/b451c6003abf6fcd12a4fb28056e5710b3befc1f)
- (f4977d5) [Why do I get `application.css not in asset pipeline` in prod…](https://github.com/rails/cssbundling-rails/commit/f4977d5514f097c3bc9884685e42e926a032e3a5)
- (f264e3f) [Fix for foreman existence test in rbenv installs. (#91)](https://github.com/rails/cssbundling-rails/commit/f264e3ff23e9850228bfddf245d76167ebfd25c7)
- (835f305) [Bump nokogiri](https://github.com/rails/cssbundling-rails/commit/835f3058c034f58700b11d34392e59f994f7e37d)
- (6b91d50) [Bump version for 1.1.1](https://github.com/rails/cssbundling-rails/commit/6b91d5069064458df7d0d9a54eeeb16f78e57500)


</details>